public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -224,24 +234,28 @@ export function FolderPicker(props: Props) {
   const createNewFolder = useCallback(
     async (folderName: string) => {
       if (folderWarning?.warningCondition(folderName)) {
+        reportInteraction('folder_picker_create_folder_validation_fail', { location });
         return false;
       }
+
       const newFolder = await createFolder({ title: folderName });
       let folder: SelectableValue<string> = { value: '', label: 'Not created' };
 
       if (newFolder.uid) {
+        reportInteraction('folder_picker_create_folder_success', { location });
         appEvents.emit(AppEvents.alertSuccess, ['Folder Created', 'OK']);
         folder = { value: newFolder.uid, label: newFolder.title };
 
         setFolder(newFolder);
         onFolderChange(folder, { action: 'create-option', option: folder });
       } else {
+        reportInteraction('folder_picker_create_folder_fail', { location });
```

Comment:
For analytics purposes when we want to see that data, it is slightly easier to have the same event with a prop that says whether the creation failed or succeeded 😉 
That means querying only one table from BigQuery instead of two. 
But it is also possible to join the tables if you see a big advantage in keeping the events separate 😊

Reply:
Exactly the type of feedback I was looking for - thanks @yaelleC. 

If we call the event like `folder_picker_create_folder`, do you think it is better to have a property like `status: fail/success/failedValidation`, or separate exclusive booleans like `failed: true` / `succeeded: true` `failedValidation: true` ?

Reply:
I would go for the former, just the one prop as they are mutually exclusive